test/python/transpiler/test_template_matching.py

```diff
@@ -657,6 +657,61 @@ def test_naming_clash_multiparameter(self):
 
         self.assertEqual(circuit_out, expected)
 
+    def test_template_match_subcircuit(self):
+        """Test that the template matching works and correctly replaces a template if there is a
+        naming clash between it and the circuit.  This should include binding a partial match with a
+        parameter.**rzx_templates.rzx_templates()"""
+        
+        """
+        Test a template based on rzx_templates(["zz3"]),
```

Comment:
Can you just directly import this template instead of rewriting the code?

Reply:
I import the template zz3 in the next version.